server/setup/csrf.ts

```diff
@@ -1,30 +1,38 @@
 import { fastifyPlugin } from 'fastify-plugin';
-import { verifyRequestOrigin } from 'lucia';
 
-import { ServerInstance } from './server';
-import { log } from '~/src/utils/log';
 import { config } from '~/src/utils/config';
+import { type ServerInstance } from './server';
+
+// TODO: A more generic way to allow requests from non-browser without a origin header
+const noOriginEndpoints = ['/v1/global-registry/inbound'];
```

Comment:
Maybe use empty array in FST as `'/v1/global-registry/inbound'` is GS1 specific.